launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -25,9 +25,9 @@
 
 import static org.apache.spark.launcher.CommandBuilderUtils.*;
 
-/**
+/** 
  * Launcher for Spark applications.
- * <p/>
```

Comment:
This was rejected by JDK8. I thought that the first line was treated differently so I dropped the spurious <p>.  Without this fix it is not possible to build the docs or publish locally so it was a serious issue for me.